packages/react-conformance-griffel/tsconfig.lib.json

```diff
@@ -5,7 +5,7 @@
     "lib": ["DOM", "ES2019"],
     "outDir": "dist",
     "declaration": true,
-    "types": ["static-assets", "environment", "jest"],
+    "types": ["static-assets", "environment", "jest", "node"],
```

Comment:
I had to add this to make the build pass after enzyme types and their indirect node type leakage were removed